backend/src/packages/drivers/driver.service.ts

```diff
@@ -79,20 +82,28 @@ class DriverService implements IService {
     return driver ? DriverEntity.initialize(driver).toObject() : null;
   }
 
-  public async findAllByBusinessId(
-    businessId: number,
-  ): Promise<DriverGetAllResponseDto> {
-    const items = await this.driverRepository.findAllByBusinessId(businessId);
+  public async findAllByBusinessId({
+    businessId,
+    query,
+  }: DriverGetDriversPagePayload &
+    DriverBusinessIdPayload): Promise<DriverGetAllResponseDto> {
+    const items = await this.driverRepository.findAllByBusinessId(
+      businessId,
+      query,
+    );
+    const total = await this.driverRepository.getTotal(businessId);
 
     return {
-      items: items.map((it) => it.toObject()),
+      items: items.map((item) => convertToDriverUser(item)),
+      total,
     };
   }
 
   public async create({
     payload,
     businessId,
-  }: DriverAddPayload): Promise<DriverAddResponseWithGroup> {
+  }: DriverAddPayload &
+    DriverBusinessIdPayload): Promise<DriverAddResponseWithGroup> {
```

Comment:
don't we have such type in shared/be ? if no, you need to create it

Reply:
I don’t think so, we use this type just here